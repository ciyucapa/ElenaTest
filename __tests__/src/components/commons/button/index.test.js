import React from 'react';
import {create} from 'react-test-renderer';

import Button from '../../../../../src/components/commons/button';

describe('Button component', () => {
    test('render button component', () => {
        const tree = create(
            <Button
                disabled
                onClick={() => {}}
                text="button title"
            />,
        ).toJSON();
        expect(tree).toMatchSnapshot();
    });
});