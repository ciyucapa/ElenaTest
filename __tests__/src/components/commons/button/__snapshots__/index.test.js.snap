// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button component render button component 1`] = `
<button
  className="sc-iBPTik xaJDg"
  disabled={false}
  onClick={[Function]}
>
  button title
</button>
`;
